1-development/list_posts_by_rating/src/main/java/com/mixfar/posts/utils/enums/ErrorCode.java

```diff
@@ -0,0 +1,30 @@
+package com.mixfar.posts.utils.enums;
+
+public enum ErrorCode {
```

Comment:
Why did you choose this name for the enum? Are you only using it in error situations?

Reply:
No. It´s a bad naming of the enum, it's should be something more generic like `StatusCode`